content/docs/idp/group-claims/aro/_index.md

```diff
@@ -155,9 +155,30 @@ oc apply -f ./cluster-oauth-config.yaml
 
 Once the cluster authentication operator reconciles your changes (generally within a few minutes), you will be able to login to the cluster using Azure AD. In addition, the cluster OAuth provider will automatically create or update the membership of groups the user is a member of (using the group ID). The provider **does not** automatically create RoleBindings and ClusterRoleBindings for the groups that are created, you are responsible for creating those via your own processes.
 
-## 4. Grant additional permissions to individual groups
+If you have a private cluster behind a firewall, you may get an error message like the image below when you try login into the web console using the AAD option. In this case you should open a firewall rule allowing access from the cluster to `graph.microsoft.com`.
+
+![Cluster Access - Authentication Error - ](../images/auth-error.png)
+
+If you are using Azure Firewall, you can run those commands to allow this access:
+
+```bash
+az network firewall network-rule create -g $AZR_RESOURCE_GROUP -f aro-private   \
+    --collection-name 'Allow_Microsoft_Graph' --action allow --priority 100     \
+    -n 'Microsoft_Graph' --source-address '*' --protocols 'any'                 \
+    --source-addresses '*' --destination-fqdns 'graph.microsoft.com'            \
+    --destination-ports '*'
+```
 
-Once the cluster authentication operator reconciles your changes (generally within a few minutes), you will be able to login to the cluster using Azure AD. In addition, the cluster OAuth provider will automatically create or update the membership of groups the user is a member of (using the group ID).
+Now you should be able to login choosing the AAD option:
+
+![Cluster Access - AAD Login - ](../images/aad-login.png)
+
+Then inform the user you would like to use:
+
+![Cluster Access - AAD Login - ](../images/aad-credential.png)
```

Comment:
This image is kind of unclear because of all the details blanked out... Could you revise the image to blur the text rather than blanking them out?